source/slang/slang-ir-link.cpp

```diff
@@ -1519,6 +1520,146 @@ static void diagnoseUnresolvedSymbols(TargetRequest* req, DiagnosticSink* sink,
     }
 }
 
+void convertAtomicToStorageBuffer(
+    IRSpecContext* context, 
+    Dictionary<int, List<IRInst*>>& bindingToInstMapUnsorted)
+{
+    IRBuilder builder = *context->builder;
+
+    for (auto& bindingToInstList : bindingToInstMapUnsorted)
+    {
+        //I don't need a sorted list, just the max so I can make an array of uint[(maxOffset/4)+1]
```

Comment:
Please remove this comment.
It simply repeats what the code is doing.
I like to remind you that our coding style states the following,
```
Try to write comments that explain the "why" of your code more than the "what".
```

Reply:
Please remove this comment.